lib/matplotlib/collections.py

```diff
@@ -106,9 +91,65 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        Create a Collection
+        Parameters
+        ----------
+        edgecolors : default: :rc:`patch.edgecolor`
+            Edge color for each patch making up the collection.
+        facecolors : default: :rc:`patch.facecolor`
+            Face color for each patch making up the collection.
+        linewidths : default: :rc:`patch.linewidth`
+            Line width for each patch making up the collection.
+        linestyles : str or tuple, default: 'solid'
+            Valid strings are ['solid', 'dashed', 'dashdot', 'dotted', '-',
+            '--', '-.', ':']. Dash tuples should be of the form::
 
-        %(Collection)s
+                (offset, onoffseq),
+
+            where *onoffseq* is an even length tuple of on and off ink lengths
+            in points. For examples, see
+            :doc:`/gallery/lines_bars_and_markers/linestyles`.
+        capstyle : default: :rc:`patch.capstyle`
+            Style to use for capping lines for all paths in the collection.
+        joinstyle : default: :rc:`patch.joinstyle`
+            Style to use for joining lines for all paths in the collection.
+        antialiaseds : default: :rc:`patch.antialiased`
+            Whether each pach in the collection should be drawn with
+            antialiasing.
+        offsets : Tuple[float, float], default: (0, 0)
```

Comment:
Changing to your suggestion because I agree that it's more human readable, and that's what matters.

On the other hand, I did do my best to dig through downstream code branches for each of the options where I used the more precise mypy notation. So in a sense I think the argument about percision (while it applies to things like `color`), specifically doesn't apply in the cases I have here.

I'll bring it up on the next call to see whether people think that we should use machine-readable type annotations wherever we are 100% confident in their accuracy.

Reply:
> I'll bring it up on the next call

:+1: